src/main/java/athena/Parser.java

```diff
@@ -240,23 +240,46 @@ public static String parseCommandType(String commandAndDetails) {
         shortcutCommands.put("v", "view");
         shortcutCommands.put("ex", "exit");
 
-        String commandType = commandAndDetails;
+        String commandType = command;
         if (shortcutCommands.get(commandType) != null) {
             commandType = shortcutCommands.get(commandType);
         }
         return commandType;
     }
 
+    /**
+     * Parses user input for shortcut commands.
+     *
+     * @param userInput String representing command and information of task
+     * @return actual input meaning string
+     */
+    public static String parseShortcutCommand(String userInput) {
+        HashMap<String, String> shortcutCommandsWithDetails = new HashMap<String, String>();
+        shortcutCommandsWithDetails.put("l3", "list i/HIGH");
+        shortcutCommandsWithDetails.put("l2", "list i/MEDIUM");
+        shortcutCommandsWithDetails.put("l1", "list i/LOW");
+        shortcutCommandsWithDetails.put("lw", "list f/WEEK");
+        shortcutCommandsWithDetails.put("lt", "list f/TODAY");
+        shortcutCommandsWithDetails.put("lm", "list f/MONTH");
+
+        String actualInputMeaning = userInput;
+        if (shortcutCommandsWithDetails.get(actualInputMeaning) != null) {
+            actualInputMeaning = shortcutCommandsWithDetails.get(actualInputMeaning);
+        }
+        return actualInputMeaning;
+    }
+
     /**
      * Parses user input and recognises what type of command
      * and parameters the user typed.
      *
-     * @param input    String representing user input
+     * @param userInput    String representing user input
      * @param taskList Tasks list
      * @return new Command object based on what the user input is
      * @throws CommandException Exception thrown when there is an error when the user inputs a command
      */
-    public static Command parse(String input, TaskList taskList) throws CommandException {
+    public static Command parse(String userInput, TaskList taskList) throws CommandException {
+        String input = parseShortcutCommand(userInput);
```

Comment:
I think if the user inputs `l3 f/WEEK`, the shortcut won't be expanded, and then the command will fail. I think `parseCommandType` and `parseShortcutCommand` can be combined together since they look at the first word in the command.